beets/util/__init__.py

```diff
@@ -355,7 +370,7 @@ def _fsencoding():
     return encoding
 
 
-def bytestring_path(path):
+def bytestring_path(path: Bytestring) -> bytes:
```

Comment:
Should probably take `Union[Bytestring, str]`)? Or maybe even `Union[bytes, str]`? I don't think we expect any other `Bytestring` subtype here.

Reply:
This is my defined type so it already is the union. I'll change the name to something less confusing in a coming commit.